backend/danswer/document_index/interfaces.py

```diff
@@ -55,6 +55,21 @@ class DocumentMetadata:
     from_ingestion_api: bool = False
 
 
+@dataclass
+class VespaDocumentFields:
```

Comment:
shoudl we also update `UpdateRequest` to take in one of these? That way there's only one source of truth for what can be updated

Reply:
This definitely makes sense! Let me add it as a TODO to one of my linear issues because when the pruning PR is in, I believe this can probably be refactored away completely.